app/src/main/AndroidManifest.xml

```diff
@@ -2,11 +2,15 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.photostomusic">
 
+    <uses-feature android:name="android.hardware.camera"
+        android:required="true" />
```

Comment:
maybe a stretch could be uploading photos instead of requiring the camera

Reply:
Yes! It is already on my notes for stretch features